docker/scripts/common.sh

```diff
@@ -102,23 +102,25 @@ e2e_test_relayer() {
                                          --l2-relay-from-rpc-url $l2_rpc \
                                          --network-to $network_config \
                                          --network-from $network_config \
-                                         --wallet-private-key $privkey \
+                                         --keystore-file $keystore_path \
+                                         --password '' \
                                          --l2-transaction-hash $txhash; \
                         do sleep 1s; done"
 }
 
 e2e_test_op_proof(){
```

Comment:
Nitpick: while we're here, can you please insert one space between `()` and `{` to make it consistent with the rest of the functions?